web/pdf_find_controller.js

```diff
@@ -322,10 +322,9 @@ var PDFFindController = (function PDFFindControllerClosure() {
      * @param {number} index - match index.
      * @param {Array} elements - text layer div elements array.
      * @param {number} beginIdx - start index of the div array for the match.
-     * @param {number} endIdx - end index of the div array for the match.
```

Comment:
This was added in commit https://github.com/mozilla/pdf.js/pull/5552/commits/2ac7ac4678519d8fcfa0f5e703531ba71bf0859d, but never appears to actually have been used, I wonder what the intention was?